pkg/qemu/qemu_driver.go

```diff
@@ -417,11 +416,7 @@ func (a *qArgTemplateApplier) applyTemplate(qArg string) (string, error) {
 				if !ok {
 					return "", fmt.Errorf("non-string argument %+v", v)
 				}
-				addr := &net.UnixAddr{
-					Net:  "unix",
-					Name: s,
-				}
-				conn, err := net.DialUnix("unix", nil, addr)
+				conn, err := netutil.DialUnix("unix", s)
```

Comment:
Nice cleanup - assuming that our helper creates the net.UnixAddr internally (but it does not).